### ISSUE TITLE
In windows Server 2012 print map and attribute tables not working

### ISSUE THREAD

**Issue by jaitor1** [3liz/lizmap-web-client#152]
I have several maps in Windows Server 2012. Athought I have activated the options to print map and added the option to show the attribute table in the Tools tab in your QGIS plugin, in lizmap both options are not being activated in my maps.

**Comment by rldhont** [comment]
Attribute table is based on WFS. Did you test the WFS Service in QGIS to verifying that the layers are published ?

Did you take a look at the server logs ?

**Comment by jaitor1** [comment]
The layers are published, of course. In fact I can edit them using lizmap.
The logs in the admin website show nothing about this. The logs in lizmap\lizmap-web-client-master\lizmap\var\logs.db show nothing about this issue (this is where the logs shown in admin.php are being stored, I suppose). 
The log files stored in lizmap\lizmap-web-client-master\lizmap\var\log (messages.log and error.log) have very few errors (4 errors) from weeks before and nothing about this issue.

**Comment by rldhont** [comment]
Edition in lizmap is not based on WFS. Can you test the WFS GetCapabilities in QGIS ?
You're on windows, and we don't know where the problem comes from. Can you take a look at this page http://gis.stackexchange.com/questions/43843/qgis-server-internal-server-error-os-109the-pipe-has-been-ended-mod-fcgid

**Comment by jaitor1** [comment]
I do not have the error of the issue. My WFS GetCapabilities is working ok:

![sin titulo](https://cloud.githubusercontent.com/assets/12136796/7931288/b73d2e56-090f-11e5-9922-fdfb4fcaa7c5.png)

If I insert http://localhost/qgis/qgis_mapserv.fcgi.exe?&SERVICE=WFS&REQUEST=GetCapabilities in a browser I get a XML with the same layers.

**Comment by rldhont** [comment]
QGIS-Server get an extra parameter which is map. This parameter has to be a path to a QGIS project. Can you test the URL with a map parameter ?

But I think that Attribute Table does not work because the name of your layers has parenthesis. I suggest to you to set name without parenthesis, space and accent. Use the title properties for complex name.

**Comment by jaitor1** [comment]
Yes, If no map specified, QGIS searches for .qgs files in the directory where the server executable is located. In my case if I add the map parameter I get the exact same XML.
http://localhost/qgis/qgis_mapserv.fcgi.exe?&SERVICE=WFS&REQUEST=GetCapabilities&MAP=portales_estadistica/getxo_portales_estadistica.qgs

I changed the name of all the layers to an easy (no spaces, no parenthesis) format, and used the Title field in lizmap for the complex name, as you suggested. Everything remains the same...Attribute table not being displayed (the option is not even enabled (displayed)) and the locate by layer tool list is empty. I tried with different fields and for all is the same in my case:

![sin titulo](https://cloud.githubusercontent.com/assets/12136796/7935131/b17cb8b4-092e-11e5-9dc8-9e4fe0ee6645.png)
